content/docs/examples/multicluster/split-horizon-eds/index.md

```diff
@@ -0,0 +1,377 @@
+---
+title: Multicluster with Split Horizon EDS
+description: Example of leveraging Istio's Split Horizon EDS to create a Multi-Cluster.
+weight: 85
+keywords: [kubernetes,multicluster]
+---
+
+This example demonstrates how to use a variation of the [Istio multicluster](/docs/setup/kubernetes/multicluster-install/vpn/) together with _Split Horizon EDS_ (introduced in Istio 1.1) to join two Kubernetes clusters together to allow access to remote instances of a service via the remote Istio ingress gateway. By following the steps in this example you will setup a topology of two clusters as described in the following diagram:
+
```

Comment:
I think it is confusion to say a variation of the multicluster vpn ... because you are not using VPN for your solution. :).  should we call it single mesh multicluster?